src/status_im/contexts/wallet/collectible/options/view.cljs

```diff
@@ -30,7 +30,6 @@
                                  #(rf/dispatch [:toasts/upsert
                                                 {:id              :random-id
                                                  :type            :positive
-                                                 :container-style {:bottom (when platform/android? 20)}
```

Comment:
It seems to me `(when platform/android? 20)` is not needed. cc @vkjr @J-Son89

Reply:
I'm not sure about this one.